### ISSUE TITLE
Puerto Rico weather file downloads not working in macro

### ISSUE THREAD

**Issue by mjprilliman** [NREL/SAM#467]
-Puerto Rico Solar Data macro not working
-Endpoint appears to have changed to: https://developer.nrel.gov/docs/solar/nsrdb/puerto-rico-download/
-Even after updating the endpoint the macro still does not run

**Comment by brtietz** [comment]
I was able to get the JSON query working, with the below webapis.conf file (renamed to .txt) which returns the text below that:

[webapis.txt](https://github.com/NREL/SAM/files/5599586/webapis.txt)

```
{ inputs={ email=[ bmirletz@case.edu, sam.support@nrel.gov ] names=2005 attributes=air_temperature,dhi,dni,ghi,surface_albedo,surface_pressure,wind_speed interval=60 wkt=POINT(-66.1 18.5) } metadata={ version=2.0.0 resultset={ count=1 } } status=200 errors=[  ] outputs={ downloadUrl=https://mapfiles.nrel.gov/data/solar/09175e1df78e96b6e68b2e32f3d44019.zip message=File generation in progress. An email will be sent to bmirletz@case.edu,sam.support@nrel.gov when the download is ready. } }
```

But the csv query always returns a 502, and I can't spot the error.

**Comment by brtietz** [comment]
2020.2.29 works even with the old api in webapis.conf.

**Comment by mjprilliman** [comment]
![image](https://user-images.githubusercontent.com/54449384/100290173-ed8bc900-2f37-11eb-8ee7-d3e234875e89.png)

Got the following message on the Siting Considerations macro as well

**Comment by brtietz** [comment]
I solved the siting macro in https://github.com/NREL/SAM/commit/9b971d9a743c48049dbf4557c8b358a32ec43fec, so it should work if you pull develop and then copy private.h from sam-private into the open source repo (just be careful not to commit it).

To summarize from the pull request: once I updated webapis.conf, the server no longer accepted two emails, which is what the query above was providing. Removing sam.support@nrel.gov from the macro worked to fully solve the issue. I also pushed the fix straight to develop on sam-private.
